examples/demo_fabricpath.pp

```diff
@@ -0,0 +1,82 @@
+# Manifest to demo cisco_fabricpath_global provider
+#
+# Copyright (c) 2014-2015 Cisco and/or its affiliates.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+class ciscopuppet::demo_fabricpath {
+
+  if platform_get() =~ /n(3|9)k/ {
+    warning('This platform does not support fabricpath feature')
```

Comment:
I think it may be better to test for the platforms that do support this feature, since XR would drop into the else condition.